packages/dashboard-frontend/src/pages/GetStarted/CustomWorkspaceTab/index.tsx

```diff
@@ -53,7 +53,7 @@ export class CustomWorkspaceTab extends React.PureComponent<Props, State> {
 
     const devfile = this.buildInitialDevfile();
     const storageType = isCheDevfile(devfile) ? attributesToType(devfile.attributes) : 'persistent';
-    const workspaceName = devfile.metadata.name ? devfile.metadata.name || '' : '';
+    const workspaceName = devfile.metadata?.name ? devfile.metadata.name || '' : '';
```

Comment:
'' is something we should not di here.... Should we?

Reply:
I have removed it